content/getting-started/accounts.md

```diff
@@ -16,4 +16,6 @@ You will need a Cloud Foundry account before continuing. To get an account:
 
 ## Resetting your password
 
-Do you have a Cloud Foundry account, but don't know the password? Go to https://login.cloud.gov/forgot_password to reset it. You can also change your password from the command line with `cf passwd`.
+If you log into Cloud Foundry using your agency's account system, you can reset your password using your agency's normal process.
+
+If you log in with a Cloud Foundry account that has its own password (including `ORGNAME_deployer` accounts), you can reset your password by going to https://login.cloud.gov/forgot_password. You can also change your password from the command line with `cf passwd`.
```

Comment:
Unfortunately it's actually not possible to set your password from the command-line unless you're an admin! We should be removing the latter sentence. (Ack, I had no idea this was out there; explains a lot!)